bears/shell/ShellCheckBear.py

```diff
@@ -14,6 +16,7 @@ class ShellCheckBear:
     """
 
     LANGUAGES = {'sh', 'bash', 'ksh', 'dash'}
+    REQUIREMENTS = {DistributionRequirement(apt_get='shellcheck')}
```

Comment:
You can now use [AnyOneOfRequirements](https://gitlab.com/coala/package_manager/blob/master/dependency_management/requirements/AnyOneOfRequirements.py) to wrap these two requirements in an OR statement.